Content.Server/Shuttles/Components/ShuttleComponent.cs

```diff
@@ -56,15 +56,6 @@ public sealed partial class ShuttleComponent : Component
         [ViewVariables]
         public DirectionFlag ThrustDirections = DirectionFlag.None;
 
-        /// <summary>
-        /// Damping applied to the shuttle's physics component when not in FTL.
-        /// </summary>
-        [DataField]
-        public float LinearDamping = 0.05f;
-
-        [DataField]
-        public float AngularDamping = 0.05f;
```

Comment:
There're like four sets of Linear/AngularDamping which were preserved on Moff compared to upstream for some reason.